recipes.js

```diff
@@ -1,11 +1,82 @@
 const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js')
+const data = require('./data');
 
-mongoose.connect('mongodb://localhost/recipeApp')
+/****************************************
+ * Tell mongoose to connect to my mongoDB
+ ****************************************/
+mongoose
+  .connect('mongodb://localhost/recipeApp')
   .then(() => {
-    console.log('Connected to Mongo!')
-  }).catch(err => {
-    console.error('Error connecting to mongo', err)
+    console.log('Connected to Mongo!');
+  })
+  .catch(err => {
+    console.error('Error connecting to mongo', err);
   });
 
+/***********************
+ * Create Schema
+ ***********************/
+const recipeSchema = new mongoose.Schema({
+  title: { type: String, required: true, unique: true },
+  level: String,
+  ingredients: [String],
+  cuisine: String,
+  dishType: String,
```

Comment:
forgot to set enum values 
`
dishType:     { type: String, enum: ['Breakfast', 'Dish', 'Snack', 'Drink', 'Dessert', 'Other']},
`